kernel/packages/decentraland-ecs/src/decentraland/Components.ts

```diff
@@ -639,6 +639,9 @@ export class TextShape extends Shape {
   @ObservableComponent.field
   billboard: boolean = false
 
+  @ObservableComponent.field
+  visible: boolean = false
```

Comment:
Why is the default `false`, if it was previously `true`?

Reply:
Because I am stupid =D, nice catch